website/landing/components/Intro/Intro.tsx

```diff
@@ -1,156 +1,28 @@
-import content from "../../website.config.json";
-import { Box, CodeBlock, List, SandpackPreview, Text } from "../common";
+import { Box } from "../common";
 
-export const Intro: React.FC = () => {
-  const { intro, commands } = content;
+import { Examples } from "./Examples";
+import { Header } from "./Header";
+import { SandpackExampleProvider } from "./SandpackExample";
+import { LayoutExampleProvider } from "./Sections/LayoutContext";
 
+export const Intro: React.FC = () => {
   return (
-    <Box
-      as="section"
-      css={{
-        alignItems: "center",
-        display: "flex",
-        flexDirection: "column",
-        overflow: "hidden",
-        padding: "200px 16px 0",
-        width: "100%",
-      }}
-    >
+    <SandpackExampleProvider>
       <Box
```

Comment:
Maybe we could use the section wrapper that is shared between the other components. It handles the layout so it's always centered on bigger screens. The content is misaligned on bigger breakpoints, see:

![rc3qm-3001 pitcher-staging csb dev_](https://user-images.githubusercontent.com/24959348/143028510-33019e50-ee28-44e0-ac16-5dd05dc4bd3d.png)